package.json

```diff
@@ -52,6 +52,7 @@
     "webpack": "^1.8.3"
   },
   "dependencies": {
+    "moment": "^2.10.2",
```

Comment:
I still want us to normalize the `dependencies` vs. `devDependencies` here. I'm unclear what the pattern is currently.